test_scripts/Smoke/API/029_OnDriverDistraction_PositiveCase.lua

```diff
@@ -0,0 +1,58 @@
+---------------------------------------------------------------------------------------------------
+-- User story: Smoke
+-- Use case: OnDriverDistraction
+-- Item: Happy path
+--
+-- Requirement summary:
+-- [OnDriverDistraction] SUCCESS: getting SUCCESS:UI.OnDriverDistraction()
+--
+-- Description:
+-- Mobile application sends valid OnDriverDistraction request and gets UI.OnDriverDistraction "SUCCESS"
+-- response from HMI
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and activated on SDL
+-- c. appID is currently in Background, Full or Limited HMI level
+
+-- Steps:
+-- appID requests OnDriverDistraction with valid parameters
+
+-- Expected:
+-- SDL validates parameters of the request
+-- SDL checks if UI interface is available on HMI
+-- SDL checks if OnDriverDistraction is allowed by Policies
+-- SDL checks if all parameters are allowed by Policies
+-- SDL transfers the UI part of request with allowed parameters to HMI
```

Comment:
@GetmanetsIrina this is a notification so SDL transfers nothing to HMI. This line could be removed